### DISCUSSION TITLE
Any experience with making GNN gym environments?

### DISCUSSION THREAD

**Discussion by antoni4040** [pyg-team/pytorch_geometric#4498] [Q&A]
(I'm probably asking too many silly questions, sorry about that)
I'm wondering if anyone has any experience with GNNs and openai gym environments. I'm trying to do reinforcement learning on graphs and having to write the algorithms instead of just making a gym environment is not very efficient. The variable action space (having to do with the dot product of edges, thus one graph can have 10 possible actions and another 20 etc) and the custom policy make me think that maybe it's not possible yet, but if anyone has managed it I would like to know more.

**Comment by rusty1s** [comment]
I do not have any experience with it (sorry about that), but I would be interested in learning more about how you would like to combine OpenAI Gym with PyG and which challenges you see regarding this.
